src/SonarQube.Common/MutexWrapper.cs

```diff
@@ -35,6 +37,11 @@ public MutexWrapper(string name)
         public MutexWrapper(string name, TimeSpan acquireTimeout)
         {
             mutex = new Mutex(false, name);
+            var allowEveryoneRule = new MutexAccessRule(new SecurityIdentifier(WellKnownSidType.WorldSid, null),
```

Comment:
A comment explaining the scenario would be useful e.g. "Concurrent builds could be run under different user accounts, so we need to allow all users to wait on the mutex"

Reply:
I just wanted to have an artifact to provide to the user to test whether that's fixing the issue but it looks like I cannot have any artifact so I'll fix the build first.